src/plugins/discover/public/application/apps/main/components/sidebar/discover_field_details.test.tsx

```diff
@@ -47,37 +48,9 @@ describe('discover sidebar field details', function () {
       aggregatable: true,
       readFromDocValues: true,
     });
-    const comp = mountComponent(visualizableField);
-    expect(findTestSubject(comp, 'fieldVisualize-bytes')).toBeTruthy();
-  });
-
-  it('should disable the visualize link for an _id field', function () {
-    const conflictField = new IndexPatternField({
-      name: '_id',
-      type: 'string',
-      esTypes: ['_id'],
-      count: 0,
-      scripted: false,
-      searchable: true,
-      aggregatable: true,
-      readFromDocValues: true,
-    });
-    const comp = mountComponent(conflictField);
-    expect(findTestSubject(comp, 'fieldVisualize-_id')).toEqual({});
-  });
-
-  it('should disable the visualize link for an unknown field', function () {
-    const unknownField = new IndexPatternField({
-      name: 'test',
-      type: 'unknown',
-      esTypes: ['double'],
-      count: 0,
-      scripted: false,
-      searchable: true,
-      aggregatable: true,
-      readFromDocValues: true,
-    });
-    const comp = mountComponent(unknownField);
-    expect(findTestSubject(comp, 'fieldVisualize-test')).toEqual({});
+    const component = mountComponent(visualizableField);
+    const onAddButton = findTestSubject(component, 'onAddFilterButton');
+    onAddButton.simulate('click');
+    expect(onAddFilter).toHaveBeenCalledWith('_exists_', visualizableField.name, '+');
```

Comment:
:information_source: This test got moved from the now deleted "Footer" component.